README.md

```diff
@@ -8,25 +8,25 @@
 
 http://www.openebs.io/
  
-OpenEBS enables the use of containers for mission critical, persistent workloads.  OpenEBS is containerized storage and related storage services.   
+**OpenEBS** enables the use of containers for mission critical, persistent workloads. OpenEBS is containerized storage and related storage services.
  
-OpenEBS allows you to treat your persistent workload containers, such as DBs on containers, just like other containers.  OpenEBS itself is deployed as just another container on your host, and enables storage services that can be designated on a per pod, application, cluster or container level, including:   
+**OpenEBS** allows you to treat your persistent workload containers, such as DBs on containers, just like other containers. OpenEBS itself is deployed as just another container on your host, and enables storage services that can be designated on a per pod, application, cluster or container level, including:
 - Data persistence across nodes, dramatically reducing time spent rebuilding Cassandra rings for example
```

Comment:
There is no example mentioned here. @kmova any suggestions what to add. 
I am going through the docs to weed out any last errors that are there in the documentation

Reply:
Resolved.